404.html

```diff
@@ -20,4 +20,4 @@ <h1>Error <span class="errorcode">404</span></h1>
          <p class="output">Please try <a href="/index.html">this link</a> or go back</p>
          <p class="output">Good luck</p>
     </div>
-</body>
\ No newline at end of file
+</body>
```

Comment:
Please close the HTML tag (hint: </html>)

Reply:
@Shayiz  Please resolve ASAP